test/cache.js

```diff
@@ -164,7 +162,7 @@ test('supports request streaming', t => {
     let buf = []
     let bufLen = 0
     res.body.on('data', d => { buf.push(d); bufLen += d.length })
-    return finished(res.body).then(() => {
+    return res.body.promise().then(() => {
```

Comment:
Minor nit: can remove the `data` handler and the buf/bufLen vars, and the `Buffer.concat()` call, and just do:

```js
res.body.concat().then(data => {
  t.deepEqual(data, CONENT)
  // ...
})
```